graphqlapi/src/main/java/io/stargate/graphql/web/models/GraphqlJsonBody.java

```diff
@@ -32,4 +32,18 @@ public void setOperationName(String operationName) {
   public void setVariables(Map<String, Object> variables) {
     this.variables = variables;
   }
+
+  @Override
+  public String toString() {
```

Comment:
so this class does not exists in the GraphQL API itself? Strange that we had to build it..

Reply:
graphql-java focuses on the core features, it's agnostic to the transport, I think that's why. The closest concept is `graphql.ExecutionInput`, but it's already in object form.

And for context, there is also a higher-level GraphQL servlet library. We used to use it, but decided to drop it for various reasons, see stargate/stargate#405.

Reply:
Good to know, resolving.